packages/smooth_app/lib/pages/product/edit_ocr/edit_ocr_image.dart

```diff
@@ -149,6 +157,26 @@ class EditOCRImageWidget extends StatelessWidget {
       ),
     );
   }
+
+  Future<void> _openOutdatedPictureExplanations(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    return showSmoothModalSheet<void>(
+      context: context,
+      builder: (BuildContext context) {
+        return SmoothModalSheet(
+          title: appLocalizations.product_image_outdated_explanations_title,
+          prefixIndicator: true,
```

Comment:
I would somehow put the outdated icon here.

Reply:
For now, the header of a modal sheet can't have an icon.

But now that we have modal sheets for Help, I think I will add a better layout for the content (including that kind of image)

Reply:
> For now, the header of a modal sheet can't have an icon.

What about the starting bold dot?
![Screenshot_20250110_100206_Chrome](https://github.com/user-attachments/assets/0e6a37d0-5aba-4aab-af74-861665ea4942)

Reply:
In the "new" design, modal sheets only have this indicator.
Hence, this is a boolean.

Reply:
That's how the "new" design may be improved, using as an indicator: a dot, nothing, or a specified icon.